app/views/games/index.html.erb

```diff
@@ -4,7 +4,8 @@
 <% @games.each do |game| %>
   <% if game.open_seat? %>
```

Comment:
Check the master branch on github :). Oh wow LOL I just noticed a mistake I made in the games controller. In the games model, it's self.list_available_games, but the controller is calling open_seats. Should be @game = Game.list_available_games. Forgot to change D: